taskfiles/deps.yaml

```
@@ -63,15 +67,14 @@ tasks:
         >> "{{.G_DEPS_CORE_CHECKSUM_FILE}}"
 
   log-viewer:
-    sources:
-      - "{{.G_YSCOPE_LOG_VIEWER_CHECKSUM_FILE}}"
-    generates: ["{{.G_DEPS_LOG_VIEWER_CHECKSUM_FILE}}"]
-    deps: ["all-internal-deps"]
     cmds:
-      - >-
-        cat
-        "{{.G_YSCOPE_LOG_VIEWER_CHECKSUM_FILE}}"
-        >> "{{.G_DEPS_LOG_VIEWER_CHECKSUM_FILE}}"
+      - task: "yscope-dev-utils:remote:download-and-extract-tar"
+        vars:
+          CHECKSUM_FILE: "{{.G_DEPS_LOG_VIEWER_CHECKSUM_FILE}}"
+          FILE_SHA256: "de0600b505545f1bceb7ff5725941035f7c1dc875d08249d9b6d679e3ba77f26"
+          OUTPUT_DIR: "{{.G_LOG_VIEWER_WEBUI_SRC_DIR}}/yscope-log-viewer"
+          URL: "https://github.com/y-scope/yscope-log-viewer/archive/969ff35.tar.gz"
+          TAR_FILE: "{{.G_BUILD_DIR}}/yscope-log-viewer.tar.gz"
```

shall we alphabetize?
```suggestion
          TAR_FILE: "{{.G_BUILD_DIR}}/yscope-log-viewer.tar.gz"
          URL: "https://github.com/y-scope/yscope-log-viewer/archive/969ff35.tar.gz"
```